mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -930,6 +930,23 @@ public void addingMessageShouldFireQuotaUpdateEvent() throws Exception {
             .isNotEmpty();
     }
 
+    @Test
+    public void deleteMailboxShouldFireQuotaUpdateEvent() throws Exception {
+        Assume.assumeTrue(mailboxManager.hasCapability(MailboxCapabilities.Quota));
+        session = mailboxManager.createSystemSession(USER_1);
+
+        EventCollector listener = new EventCollector();
+        mailboxManager.addGlobalListener(listener, session);
+
+        MailboxPath inbox = MailboxPath.inbox(session);
+        mailboxManager.createMailbox(inbox, session);
+        mailboxManager.deleteMailbox(inbox, session);
+
+        assertThat(listener.getEvents())
+            .filteredOn(event -> event instanceof MailboxListener.MailboxDeletion)
```

Comment:
`ShouldFireQuotaUpdateEvent` and you are looking for `MailboxDeletion` events? Feels wrong....

Maybe a naming issue....